### PR TITLE
GH-509 Additional YAxis Types

### LINKED ISSUE #509: Chart YAxis add support for exponential and log scales

### PR DESCRIPTION
**Pull request by GordonSmith**
Add "time" as an option to YAxis.

Fixes #509

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@anmoljagetia, @mlzummo Please Review

File: src/chart/XYAxis.js [hunk 1]
```diff
@@ -10,24 +10,25 @@
         SVGWidget.call(this);
         this._drawStartPos = "origin";
 
-        this._dateParser = d3.time.format("%Y-%m-%d").parse;
+        this._dateParserData = d3.time.format("%Y-%m-%d").parse;
+        this._dateParserValue = d3.time.format("%Y-%m-%d").parse;
```
Comment (mlzummo) [inline src/chart/XYAxis.js]: how come this is here and two of the same kind of object? do these return 0000-00-00 as default for a placeholder?

Comment (GordonSmith) [inline src/chart/XYAxis.js]: The xAxis and yAxis might get dates in two different formats.

**Comment by GordonSmith** [comment]
@anmoljagetia, @mlzummo Please Re-Review

**Comment by mlzummo** [comment]
what was changed?

**Comment by GordonSmith** [comment]
Added "stacked" option to Columns

**Comment by anmoljagetia** [comment]
Looks good.

**Pull request merged**